src/obstacle_avoidance/src/MyNode-devel.cpp

```diff
@@ -0,0 +1,99 @@
+/*
+ * Created By: Chris Heathe
+ * Created On:  October 28, 2017
+ * Description: A node that will theoretically interpret lidar data,
+ *              and help Olaf navigate through a hole in a row of cones
+ *              Stress on *theoretically*
+ */
+
+#include <MyNode.h>
+
+MyClass::MyClass(int argc, char **argv, std::string node_name) {
+    // Setup NodeHandles
+    ros::init(argc, argv, node_name);
+    ros::NodeHandle nh;
+    ros::NodeHandle private_nh("~");
+
+    // Obtains character from the parameter server (or launch file), sets '!' as default
+    std::string parameter_name = "my_node/character";
+    std::string default_character = "!";
+    SB_getParam(nh, parameter_name, suffix, default_character);
```

Comment:
We don't need to get this parameter for the obstacle avoidance node (so this section can be removed). 

We use parameters to set options on nodes when we launch them, rather then having to recompile every time we want to tune something. They can be set via launch files. See the `my_node.launch` file for an example of how we set the parameter for this node in particular.